lib/src/component/prop_mixins.dart

```diff
@@ -267,24 +267,15 @@ abstract class $UbiquitousDomPropsMixin {
 abstract class _$UbiquitousDomPropsMixin {
   Map get props;
 
-  /// A cache for the MapView used for [aria].
-  @Accessor(doNotGenerate: true)
-  AriaPropsMapView _aria;
-
-  /// A cache for the MapView used for [dom].
-  @Accessor(doNotGenerate: true)
-  DomProps _dom;
-
   /// A view into this map that can be used to access `aria-` props, for convenience.
   ///
   /// Example:
   ///
   ///     (Button()
   ///       ..aria.controls = 'my_popover'
   ///     )('Open popover')
-  AriaPropsMixin get aria {
-    return _aria ??= AriaPropsMapView(props);
-  }
+  @Accessor(doNotGenerate: true)
+  late final AriaPropsMixin aria = AriaPropsMapView(props);
```

Comment:
These could've stayed as-is, but this was just the perfect use-case for `late final` 🙂